generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -32,27 +35,33 @@
       MethodHandles.lookup().lookupClass().getName());
 
   private final String[] argv;
-  private final Path generatedFile;
+  private final File generatedFile;
```

Comment:
Hmm. There's not much point of using jimfs if it won't let us use an in-memory filesystem anyway.  Please modify this to either be all in memory or all on disk (for tests).